src/Pages/Basket/QuantityForm/QuantityForm.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import './QuantityForm.scss';
+
+class QuantityForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      amount: this.props.item[0].amount,
+    };
+  }
+
+  handleClickControlButton = e => {
+    const { order_item_id } = this.props.item[0];
+    if (e.target.name === 'cancle') {
+      this.props.openQuantityForm(order_item_id);
+    } else {
+      this.props.requestModifyQuantity(order_item_id, this.state.amount);
+      this.props.openQuantityForm(order_item_id);
+    }
+  };
+
+  handleClickCountButton = e => {
+    if (e.target.name === 'minus') {
+      if (this.state.amount <= 0) return;
+
+      this.setState(prev => prev.amount--);
+    } else {
+      this.setState(prev => prev.amount++);
+    }
+  };
```

Comment:
지금 형태의 업데이트는 state를 직접 업데이트 하는 방식이기 때문에 불변성을 지켜주면서 업데이트 하는 것을 권장하는 react의 정책을 위반합니다. 다음과 같이 얕은 복사를 통해 새 객체를 생성하는 방식으로 업데이트 해주세요
```suggestion
  handleClickCountButton = e => {
    if (e.target.name === 'minus') {
      if (this.state.amount <= 0) return;

      this.setState(prev => ({ ...prev, amount: prev.amount - 1 }));
    } else {
      this.setState(prev => ({ ...prev, amount: prev.amount + 1 }));
    }
  };
```